src/test/resources/data/InstructorFeedbackEditCopyTest.json

```diff
@@ -317,6 +317,29 @@
         "sectionLevel": {},
         "sessionLevel": {}
       }
+    },
+    "teammates.test.instructor3": {
```

Comment:
Why exactly do you need an extra instructor here?

Reply:
I set custom privileges for this instructor to test that he is not allowed to copy sessions in course. I have tried using `instructor1d` and `instructor2e` pre-defined in the bundle but for some reason their privileges end up being the same as `instructor1` and `instructor2` who are `Co-owner`s. Maybe because they have the same `googleId`?

Reply:
Those two instructors have `canviewsessioninsection` set to `true` as configured in this test data file. That might be the cause of your failure. If that really is the case, see if modifying one of them works. Let's not add test entities unless necessary.

Reply:
Yes I have tried changing their `canviewsessioninsection` to `false` and used them to login (`instructor1d` then `instructor2`, and `instructor2e` then `instructor`, for some reason `testAccessControl` did not throw the expected `UnauthorizedAccessException`.

Reply:
> On the other hand if I modify instructor and instructor2 ( role to Custom and canviewsessioninsection to false) and use them for testAccessControl it works.

I think you would have figured out why this is the case. (I know why)
I guess just one extra instructor for very few tests (only 3 tests use this small data bundle) will not hurt much, so I will let this pass.